backend/session/logind.c

```diff
@@ -70,28 +69,30 @@ static int logind_take_device(struct wlr_session *base, const char *path) {
 		session->path, "org.freedesktop.login1.Session", "TakeDevice",
 		&error, &msg, "uu", major(st.st_rdev), minor(st.st_rdev));
 	if (ret < 0) {
-		wlr_log(WLR_ERROR, "Failed to take device '%s': %s", path, error.message);
-		goto error;
+		wlr_log(WLR_ERROR, "Failed to take device '%s': %s", path,
+			error.message);
+		goto out;
```

Comment:
```fd``` is not defined if this goto is triggered, but it is needed for the return.

Reply:
Yeah, just pushed a fix